packages/sewing-kit-koa/src/middleware.ts

```diff
@@ -18,22 +18,28 @@ export interface Options {
   serveAssets?: boolean;
   manifestPath?: string;
   caching?: boolean;
+  key?: symbol | string;
 }
 
 const ASSETS = Symbol('assets');
 
-export function getAssets(ctx: Context): Assets {
-  return (ctx.state as any)[ASSETS];
+export function getAssets(ctx: Context, key: string | symbol = ASSETS): Assets {
```

Comment:
This shouldn't allow arbitrary `key` access on context as it's a generic koa context object and this would allow you to accidentally overwrite or access values across middleware. If the goal is to allow multiple different types of asset access by key within the object then `ctx[ASSETS]` will need to be changed to a Map/Object instead that can maintain each of those. We can create a second `DEFAULT_ASSETS = Symbol('default_assets')` within that map to preserve the existing behaviour.